relay-server/src/service.rs

```diff
@@ -163,6 +158,15 @@ impl ServiceState {
         )
         .start_in(&aggregator_runtime);
 
+        let outcome_producer = OutcomeProducerService::create(
+            config.clone(),
+            upstream_relay.clone(),
+            envelope_manager.clone(),
+        )?
+        .start_in(&outcome_runtime);
+        let outcome_aggregator =
+            OutcomeAggregator::new(&config, outcome_producer.clone()).start_in(&outcome_runtime);
```

Comment:
I'm not sure whether moving the outcome producer to the end here creates any problems.